specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2021-04-01-preview/examples/ApiManagementUpdateApiOperation.json

```diff
@@ -62,7 +62,6 @@
             "representations": [
               {
                 "contentType": "text/xml",
-                "sample": "<?xml version=\"1.0\" encoding=\"utf-8\"?>\r\n<Envelope xmlns=\"http://schemas.xmlsoap.org/soap/envelope/\">\r\n  <Body>\r\n    <CancelOrder xmlns:xsi=\"http://www.w3.org/2001/XMLSchema-instance\" xmlns=\"http://tempuri.org/\">\r\n      <orderId>1</orderId>\r\n    </CancelOrder>\r\n  </Body>\r\n</Envelope>",
```

Comment:
Why remove sample?

Reply:
Because RepresentationContract in definitions.json doesn't have "sample" field in this preview.